lib/config.js

```diff
@@ -90,6 +92,7 @@ module.exports = {
  * @property {string} apiServerUrl API server URL that the SDK will use for requests.
  * @property {string} [apiVersion] Version of the RewardOps API to use for requests.
  *   This affects the SDK methods available and the API baseUrl.
+ * @property {string} piiServerUrl Geographic-specific PII storage server URL that the SDK will use for requests.
  * @property {string} [logFilePath] Path for log output file.
  * @property {boolean} [logToFile] If `true` and if {@link config.verbose} is `true`, saves log messages to file.
  * @property {number} [timeout] Timeout for HTTP requests (used by [Request]{@link https://github.com/request/request}).
```

Comment:
We should also add a `@property` for `supportedLocales` here, if we're keeping it